std/file.d

```diff
@@ -2304,6 +2308,39 @@ if (isConvertibleToString!R)
     }
 }
 
+version (Windows)
+{
+    /// File attributes that can be used on Windows usng ex. $(LREF setAttributes)
+    enum FileAttribute
+    {
+      /// A file or directory that is an archive file or directory. Applications typically use this attribute to mark files for backup or removal.
+      archive = 0x20,
```

Comment:
Why hard-code these constants instead of reusing the ones from `core.sys.windows`?